aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -315,7 +315,12 @@ void persistGlueTable(Table glueTable, Map<String, String> parameters, TableMeta
                       .build());
       // Use Optimistic locking with table version id while updating table
       if (!SET_VERSION_ID.isNoop() && lockManager == null) {
+        LOG.debug("Using optimistic locking with table version: {}", glueTable.versionId());
```

Comment:
Configuration should be logged once, not every time it is used. Can you fix this to log what you want to when the catalog is initialized?

Reply:
Thank you for your comment! Reflected in the latest commit.